eclipse-scout-core/src/text/DateFormat.js

```diff
@@ -86,11 +86,18 @@ export default class DateFormat {
     //   (e.g. MMMM) to short (e.g. M).
     this._patternDefinitions = [
       // --- Year ---
+      // This definition can _format_ dates with years with 4 or more digits.
+      // However: it is not allowed to _parse_ a date with 5 or more digits. In a future release we could allow
+      // that, but it could have an impact on backend logic, databases, etc.
```

Comment:
Can you add a link to more information about 5-digit years? (RFC, MDN, ISO standard, stackoverflow or similar)
I think this would help a future developer in understanding the reason for this code.